### PR TITLE
Feature/cash contributions

### PR DESCRIPTION
**Pull request by PushBackNeil**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -13,12 +13,16 @@
 <div class="govuk-width-container">
   <main class="govuk-main-wrapper">
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
-        <%= render "layouts/header" %>
-        <p class="notice"><%= notice %></p>
-        <p class="alert"><%= alert %></p>
-        <%= yield %>
-        <%= render "layouts/footer" %>
+      <% if request.path.to_s == '/project/cash-contribution' %>
```
Comment (gidsg) [inline app/views/layouts/application.html.erb]: We can use `project_cash_contribution_path` to resolve this route rather than a string

Comment (gidsg) [reply app/views/layouts/application.html.erb]: Doesn't seem great to have a layout rule for one page in this template. Maybe it should be a custom layout? e.g. https://guides.rubyonrails.org/layouts_and_rendering.html#the-layout-option at least we should add a TODO to review this?

Comment (PushBackNeil) [reply app/views/layouts/application.html.erb]: @gidsg I agree, this was just a quick and dirty just so I could see the layouts whilst adding content until we had flushed out all the routing, I am sure there will be other similar scenario's. I will add a TODO for this now and push a new commit up to this branch.

**Comment by PushBackNeil** [comment]
I am going to close this request, I have merged this branch in to feature/non-cash-contributions and resolved the master merge conflicts there.

**Pull request closed without merging**